raiden/connection_manager.py

```diff
@@ -286,6 +286,8 @@ def _join_partner(self, partner: Address):
             log.exception('connection manager: channel not in opened state')
         except InsufficientFunds as e:
             log.error(f'connection manager: {str(e)}')
+        except (DepositOverLimit, DepositMismatch) as e:
+            log.exception(f'connection manager: {str(e)}')
```

Comment:
Please, change this `str(e)` to use `structlog`'s kwargs, and also in the line above, like:
```
log.exception('connection manager: join_partner', _exception=e)
```
Notice `exception` is a reserved kwargs for `structlog`, that's why the `_`

Reply:
Ok then, thanks @andrevmatos .

Reply:
Also: why not put these two in the same line as `InsufficientFunds`?

Reply:
No, I wondered why you don't write: `except (InsufficientFunds, DepositOverLimit, DepositMismatch) as e:`